.eslintrc.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
         'keyword-spacing': ['error'],
         'no-throw-literal': ['error'],
         'max-statements-per-line': ['error'],
+        'max-params': ["error", 7],
```

Comment:
```suggestion
        'max-params': ['error', 7],
```

There are a few cases where functions use more than 7 parameters currently that will need to be fixed.

Reply:
Shall I make it dynamic such that it takes value for env to set maximum number of parameters?